### ISSUE TITLE
Enhance zoom-control example

### ISSUE THREAD

**Issue by OpenSteveMap** [yagajs/leaflet-ng2#138]
The zoom-control example shows at the moment `[zIndex]`and `[(display)]`, but this is not part of the zoom-control-directive.

The example for zoom-control should show instead `[zoomInText]`, `[zoomInTitle]`,  `[zoomOutText]`, `[zoomOutTitle]`.

**Comment by atd-schubert** [comment]
Add select boxes like this for position state:
```ts
properties = {
     { name: 'test', value: true, type: 'select', additional: { states: ['bottomleft', '...']} }
}
```

**Comment by atd-schubert** [comment]
Merged with #152
